lib/bolt/executor.rb

```diff
@@ -169,6 +169,7 @@ def run_task(targets, task, arguments, options = {}, &callback)
       description = options.fetch('_description', "task #{task.name}")
       log_action(description, targets) do
         notify = proc { |event| @notifier.notify(callback, event) if callback }
+        options = { '_task' => task.name }.merge(options)
```

Comment:
What happens when this is sent to orchestrator?
this should probably just set the value after the merge on the next line. We don't want to favor a `_task` key passed  in from elsewhere

Reply:
Like so?
```
        options = { '_run_as' => run_as }.merge(options) if run_as
        options = options.merge('_task' => task.name)
```

Reply:
I think just `options['_task'] = task.name` is more readable and performant